src/Core/Migration/V6_7/Migration1737430168RemoveFileTypeOfDocumentTable.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_7;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ */
+#[Package('after-sales')]
+class Migration1737430168RemoveFileTypeOfDocumentTable extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1737430168;
+    }
+
+    public function update(Connection $connection): void
+    {
+    }
+
+    public function updateDestructive(Connection $connection): void
+    {
+        $this->dropColumnIfExists($connection, 'document', 'file_type');
```

Comment:
@nguyenytran the `updateDestructive` method is not guranteed to be executed directly with the update to 6.7, but can be executed later, to allow rollback to an older version without touching the database

I would suggest that we add some useful default for that column in the `update` method here, that way we don't need to provide it in the application anymore, but the data is still valid for older application versions

CC @MalteJanz as it touches documents

Reply:
looking at this we can even put the delete in the `Update` method, because we already deprecated everything